charts/nri-kube-events/README.old.md

```diff
@@ -0,0 +1,46 @@
+# nri-kube-events
+
```

Comment:
I thought that having the old file there, it would be easier to diff/compare files. I am leaving this thread open until I finish so I don't forget to clean it.

Reply:
It is now removed.